scripts/g.extension/g.extension.py

```diff
@@ -232,6 +232,17 @@ def urlopen(url, *args, **kwargs):
     ability to define headers.
     """
     request = urlrequest.Request(url, headers=HEADERS)
+    try:
```

Comment:
We need catch this error on the top level, inside `etree_fromurl()` function, code line 381.

```python
def etree_fromurl(url):
    """Create XML element tree from a given URL"""
    try:
        file_ = urlopen(url)
    except URLError:
        gscript.fatal(
            _(
                "Download file from <{url}>,"
                " failed. File is not on the server or"
                " check your internet connection.".format(
                    url=url,
                ),
            ),
        )
    return etree.fromstring(file_.read())
```

Reply:
Thanks for the suggestion, implemented in d2a6001626396a7275d538219b5d98c480f92fe8